### ISSUE TITLE
Add href to links in the demo

### ISSUE THREAD

**Issue by renatoi** [codrops/LineMenuStyles#5]
Otherwise I cannot test how well it would work for keyboard users, since links without href won't have a tab stop.

**Comment by botelho** [comment]
Added! Cheers.

**Comment by renatoi** [comment]
awesome!
